### DISCUSSION TITLE
Automation update row with Today's date

### DISCUSSION THREAD

**Discussion by alexmanwin1010** [Budibase/budibase#9197] [General]
Hi, how do you update a row during an automation with today's date, or the date the automation is run?

**Comment by melohagan** [comment]
Hey @alexmanwin1010 

Have you tried the following binding? `{{ date now "yyyy-MM-dd HH:mm:ss" }}`

**Reply by alexmanwin1010** [reply]
yes, i still get an error when trying to update:
<img width="416" alt="Screen Shot 2023-01-03 at 5 20 34 PM" src="https://user-images.githubusercontent.com/121869196/210397652-51f9f215-43b8-4e63-b750-cb1443facec7.png">

**Reply by melohagan** [reply]
Have you selected `Binding` in the Value/Binding dropdown?

**Reply by alexmanwin1010** [reply]
yes :(  here's a bigger view screenshot of the process and automation test, any help is greatly appreciated! 

<img width="1041" alt="Screen Shot 2023-01-03 at 8 19 52 PM" src="https://user-images.githubusercontent.com/121869196/210426313-ffffa66c-10ad-4ef9-bd68-3eeb4e7380b9.png">

**Reply by smurraystrata** [reply]
I am using {{ now }}:
![image](https://user-images.githubusercontent.com/120056389/210436084-ed2429d9-23c0-47cb-9257-704cf385c186.png)
And it shows the date and time:
![image](https://user-images.githubusercontent.com/120056389/210436166-0d7c9b5e-545e-461a-90de-a652b6f06fe2.png)
